src/Arcus.Messaging.Tests.Integration/Arcus.Messaging.Tests.Integration.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Arcus.Testing.Logging.Xunit" Version="1.1.0" />
     <PackageReference Include="Arcus.Testing.Security.Providers.InMemory" Version="1.1.0" />
     <PackageReference Include="Arcus.Testing.Storage.Blob" Version="1.2.0" />
+    <PackageReference Include="Arcus.Testing.Core" Version="1.3.0" />
```

Comment:
Will eventually also be removed, as it will become a transient dependency once we use the Arcus.Testing package for Service bus.